admin_guide/allocating_node_resources.adoc

```diff
@@ -155,6 +164,107 @@ $ curl <certificate details> https://<master>/api/v1/nodes/cluster.node22/proxy/
 
 See xref:../rest_api/index.adoc[REST API Overview] for more details about certificate details.
 
+[[node-enforcement]]
+== Node enforcement
+
+The node is able to enforce the total amout of compute resources that pods
+may consume based on the configured allocatable value.  This feature significantly
+improves the reliability of the node by preventing end-user pods from starving
+system services (i.e. container runtime, node agent, etc.) of required compute
+resources.  It is strongly encouraged that administrators set aside system
+resources based on the desired density of their workload in order to improve
+system reliability.
+
+The node enforces allocatable constraints via a new cgroup hierarchy
+that enforces quality of service.  All end-user pods are launched in a
+dedicated cgroup hierarchy separate from system daemons.
+
+To configure this ability, the following kubelet arguments are provided.
+
+.Node Cgroup Settings
+====
+[source,yaml]
+----
+kubeletArguments:
+  cgroups-per-qos:
+    - "true" <1>
+  cgroup-driver:
+    - "systemd" <2>
+  enforce-node-allocatable:
+    - "pods" <3>
+----
+<1> Enable or disable the new cgroup hierarchy managed by the node.  Any change
+of this setting requires a full drain of the node.  It is strongly encouraged
```

Comment:
s/It is strongly encouraged that users not change this value."/We recommend that users do not change this value.